src/components/custombutton-large.tsx

```diff
@@ -1,16 +1,16 @@
-import React from "react";
+import { HTMLAttributes } from "react";
 
-interface CustomButtonProps {
+interface CustomButtonProps extends HTMLAttributes<HTMLButtonElement> {
   onClick: React.MouseEventHandler<HTMLButtonElement>;
   text: string;
 }
 
-const CustomButton: React.FC<CustomButtonProps> = ({ onClick, text }) => {
+const CustomButton = ({ onClick, text }: CustomButtonProps) => {
```

Comment:
This is almost right, but in order to get all the additional HTML Button attributes we need something this:
```ts
const CustomButton = ({ onClick, text, ...props }: CustomButtonProps) => {
```
Here, the spread operator gets all of the props passed (other than `onClick` and `text`). Then you can pass these props to the `button` below.